OmniKit/PumpManager/PodComms.swift

```diff
@@ -67,16 +67,17 @@ class PodComms: CustomDebugStringConvertible {
             throw PodCommsError.podFault(fault: fault)
         }
         
-        guard let config = response.messageBlocks[0] as? VersionResponse else
-        {
+        guard let config = response.messageBlocks[0] as? VersionResponse,
+            config.isAssignAddressVersionResponse == true
```

Comment:
I think this is overly defensive and not strictly necessary.

Reply:
I'm guarding against potential transport errors &/or pods in funky states which we've seen before. We should catch any none $0115 AssignAddress responses here instead of proceeding.